<launch>

<!-- imu -->

  <node pkg="provider_imu" type="provider_imu_node" name="provider_imu_node" output="screen">
    <param name="time_offset" value="-0.040" />
    <param name="port"  type="string" value="/dev/ttyACM0" />
    <param name="autostart" type="bool" value="true" />
    <param name="frameid"  type="string" value="imu" />
    <param name="autocalibrate" type="bool" value="false" />
    <param name="angular_velocity_stdev" type="double" value="0.00017" />
  </node>

</launch>
